### PR TITLE
Mark repository as deprecated

### PR DESCRIPTION
**Pull request by ferki**
This PR adds a notice to the README about its state.

@ehuelsmann: please review and let me know if there's anything to add or reword.

I decided to just simply prepend the deprecation notice, but we can also just replace the README completely if you think that's better.

### CODE REVIEW

File: README.pod [hunk 1]
```diff
@@ -1,5 +1,27 @@
...
+
+This repository is considered as a deprecated approach to collect and distribute Rex modules, and will be archived.
+
+The main problems are:
+
+=over 4
+
+=item Doesn't scale
+
+A few people must maintain all the modules, including testing and reviews, and even if they are not the authors. That proved to be an unrealistic expectation.
```
Comment (ehuelsmann) [inline README.pod]: s/ and / /
?

File: README.pod [hunk 2]
```diff
@@ -1,5 +1,27 @@
...
+
+=item Disconnects authors from their code
+
+Once a module is merged, the author loses direct maintainership of it, since they can only interact with it via pull requests. They could be granted access to directly manage this repo, therefore all modules, but that is probably not anyone wants.
```
Comment (ehuelsmann) [inline README.pod]: s/not anyone/not what anyone/
?

**Comment by ehuelsmann** [comment]
@ferki would it make sense to point people to a document (from the Perl community, not on our own site) which explains how to upload Perl modules to PAUSE/CPAN?

Another item: currently rexify has a way to "locally install" a module. Maybe we need to declare whether that will still work, but just download from CPAN or that it won't work anymore, or...?

**Review by ehuelsmann — CHANGES_REQUESTED** [review]
As per my comments, I think we'll want a few changes, potentially after discussion first.

File: README.pod (hunk 1 above)
Comment (ferki) [reply README.pod]: Fixed

File: README.pod (hunk 2 above)
Comment (ferki) [reply README.pod]: Fixed

**Comment by ferki** [comment]
> @ferki would it make sense to point people to a document (from the Perl community, not on our own site) which explains how to upload Perl modules to PAUSE/CPAN?

I've slightly reworded that part and included links.

> Another item: currently rexify has a way to "locally install" a module. Maybe we need to declare whether that will still work, but just download from CPAN or that it won't work anymore, or...?

Well, no code changes in that regard, so that works as before. It is already able to handle installing from CPAN and/or directly from git repos.

**Comment by ferki** [comment]
@ehuelsmann: thanks for the review, I believe I've addressed/answered all items. Additional `fixup!` commits must be squashed away upon merging ;)

**Review by ehuelsmann — APPROVED** [review]
Yup. It's fine now!

**Pull request merged**